x2b.py

```
@@ -439,64 +447,45 @@ def get_x_posts():
+    except json.JSONDecodeError:
+        print(result.stdout[:2000], file=sys.stderr)
+        raise
 
-        if len(posts) < fetch_count:
-            break
+    if not data.get("ok"):
+        raise RuntimeError("twitter-cli returned ok=false")
 
-        remaining -= len(posts)
+    # twitter-cliが要求件数を超えて返しても
+    # runあたりのハード上限を守る（ADR 0001）
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)